static/js/components/EmploymentTab.js

```diff
@@ -103,8 +91,10 @@ class EmploymentTab extends ProfileTab {
       setWorkDialogIndex,
       setWorkDialogVisibility,
     } = this.props;
-    let clone = _.cloneDeep(profile);
-    clone['work_history'] = clone['work_history'].concat(this.blankWorkHistoryEntry);
+    let blankWorkHistoryEntry = generateNewWorkHistory();
+    let clone = Object.assign({}, profile, {
+      work_history: profile.work_history.concat(blankWorkHistoryEntry)
+    });
```

Comment:
is the `let blankWorkHistoryEntry` line just for clarity? Wondering why not just:

```js
let clone = Object.assign({}, profile, {
  work_history: profile.work_history.concat(generateNewWorkHistory())
});
```